_pages/javascript/frameworks.md

```diff
@@ -62,6 +64,7 @@ Backbone.js is a JavaScript library with a RESTful JSON interface and is based o
 
 
 ## React
+{%include components/tag-suggestion.html %}
```

Comment:
Minor, and feel free to overrule me, but I think we can safely say we default to React by now. I could even see a good argument for Standard.

@hbillings I know you've had this discussion before. Any opinion?